en/docs/get-started/streaming-quick-start-guide.md

```diff
@@ -1,141 +1,103 @@
-
-
-2. Enter `Y` in the terminal to confirm that you want to proceed to install the required extensions, and then press the return key. Then the following message is displayed to indicate that the extension is installed.
-
-    ```text
-    Installing kafka ...
-    Installation completed with status: INSTALLED.
-
-    Installed missing extension 1 of 1.
-    Please restart the server.
+    insert into ProductionSumAlertStream;
     ```
 
-3. Restart the WSO2 Streaming Integrator server as instructed.
-
-
-## Step 4: Start Kafka and create a topic
-
-Let's start the Kafka server and create a Kafka topic so that the `ManageProductionStats.siddhi` application you created can publish its output to it.
+3. Save this file as `ManageStats.siddhi` in the `<SI_HOME>/wso2/server/deployment/siddhi-files` directory.
 
-To start Kafka:
+    This deploys the `ManageStats` in the Streaming Integrator. The following message appears to indicate that the Siddhi application installed successfully.
 
-1. Navigate to the `<KAFKA_HOME>` directory and start a zookeeper node by issuing the following command.
+    `INFO {org.wso2.carbon.siddhi.editor.core.internal.WorkspaceDeployer} - Siddhi App ManageStats.siddhi successfully deployed.`
 
-    `sh bin/zookeeper-server-start.sh config/zookeeper.properties`
 
-2. Navigate to the `<KAFKA_HOME>` directory and start Kafka server node by issuing the following command.
+## Step 4: Test your Siddhi application
 
-    `sh bin/kafka-server-start.sh config/server.properties`
+To test the `ManageStats` Siddhi application you created, follow the steps below.
 
-To create a Kafka topic named `total_production`:
+1. Open the `/Users/foo/sampledata.xml` file and add rows in it as follows:
 
-1. Navigate to `<KAFKA_HOME>` directory and issue the following command:
-
-    `bin/kafka-topics.sh --create --bootstrap-server localhost:9092 --replication-factor 1 --partitions 1 --topic total_production`
-
-
-## Step 5: Test your Siddhi application
-
-To test the `ManageProductionStats` Siddhi application you created, follow the steps below.
-
-1. Open the `/Users/foo/productions.csv` file and add five rows in it as follows:
-
-    `Toffee,40.0`<br/><br/>
-    `Almond cookie, 70.0`<br/><br/>
-    `Baked alaska, 30.0`<br/><br/>
-    `Toffee, 60.0`<br/><br/>
-    `Baked alaska, 20.0`
+    `    
+    <event>
+        <name>Almond cookie</name>
+        <amount>70</amount>
+    </event>
+    <event>
+        <name>Baked alaska</name>
+        <amount>30</amount>
+    </event>
+    <event>
+        <name>Toffee</name>
+        <amount>60</amount>
+    </event>`
 
     Save your changes.
 
-2. To observe the messages in the `total_production` topic, navigate to the `<KAFKA_HOME>` directory and issue the following command:
-
-    `bin/kafka-console-consumer.sh --bootstrap-server localhost:9092 --topic total_production --from-beginning`
+2. To observe the MI console output.
```

Comment:
Should this be "MI console output" OR "SI console output"?

Reply:
This shpuld be MI. the SiddhiApp is deployed in MI.